### PR TITLE
[docs] ClickHouse

### PR DESCRIPTION
**Pull request by withshubh**
#### changes
- Added clickhouse.md in datasource reference and clickhouse images in static/img

### CODE REVIEW

**Comment by shubh22** [comment]
Task linked: [CU-2r6bmz5 Clickhouse DB](https://app.clickup.com/t/2r6bmz5)

File: docs/docs/data-sources/clickhouse.md [hunk 1]
```diff
@@ -1,21 +1,214 @@
...
+To add a new ClickHouse datasource, Go to the **Datasource Manager** on the left sidebar of the app editor and click on `Add datasource` button. Select **ClickHouse** from the modal that pops up.
+
+ToolJet requires the following to connect to your ClickHouse Database:
+
+- **Username**
+- **Password**
+- **Host**
+- **Port**
+- **Database Name**
+- **Protocal**
```
Comment (stepinfwd) [inline docs/docs/data-sources/clickhouse.md:25]: protocol

File: docs/docs/data-sources/clickhouse.md [hunk 2]
```diff
@@ -1,21 +1,214 @@
...
+		date Date,
+		str String,
+		arr Array(String),
+		arr2 Array (Date)
+		arr3 Array(UInt32) ,
+		id1 UUID
+	)ENGINE=MergeTree () ORDER BY(str)
+	```
+
+	:::info
```
Comment (stepinfwd) [inline docs/docs/data-sources/clickhouse.md:78]: can remove this info

File: docs/docs/data-sources/clickhouse.md [hunk 3]
```diff
@@ -1,21 +1,214 @@
...
+
+	<div style={{textAlign: 'center'}}>
+
+	<img className="screenshot-full" src="/img/datasource-reference/clickhouse/alter.png" alt="ClickHouse SQL Statement operation" />
+
+	</div>
+
+- **SELECT WITH WHERE CLAUSE**
+	```sql
+	SELECT * FROM test array1 WHERE str='Somethingl...'  1 ALTER TABLE test_array1 UPDATE arr = (12] WHERE str='Somethingl...'
```
Comment (stepinfwd) [inline docs/docs/data-sources/clickhouse.md:102]: second query is not needed

File: docs/docs/data-sources/clickhouse.md [hunk 4]
```diff
@@ -1,21 +1,214 @@
+---
+id: clickhouse
+title: ClickHouse
+---
 
 # ClickHouse
 
-ToolJet can connect to ClickHouse databases to read and write data. 
-ClickHouse® is a column-oriented database management system (DBMS) for online analytical processing of queries (OLAP).
+ToolJet can connect to the ClickHouse databases to read and write data. ClickHouse® is a column-oriented database management system (DBMS) for online analytical processing of queries (OLAP).
```
Comment (stepinfwd) [inline docs/docs/data-sources/clickhouse.md:8]: **databases

**Review by stepinfwd — CHANGES_REQUESTED** [review]

**Review by stepinfwd — APPROVED** [review]

**Pull request merged**